tincr/io/library/genCellLibrary.tcl

```diff
@@ -1,4 +1,13 @@
-package require tincr 0.0
+package provide tincr.io.library 0.0
+package require Tcl 8.5
+package require tincr.cad.design 0.0
+package require tincr.cad.device 0.0
+package require tincr.cad.util 0.0
+
+namespace eval ::tincr:: {
+    namespace export \
+        createCellLibrary
```

Comment:
I think that consistency will benefit us in the long run, so my suggestion would be to unify both public and private methods on the same naming convention.